src/Microsoft.PackageManagement.PackageSourceListProvider/PackageList/PackageListParser.cs

```diff
@@ -140,7 +140,7 @@ public static object ConvertObjectToType(object objectToBeConverted, Type typeOf
                     }
                     else
                     {
-                        throw new ArgumentException(string.Format("'{0}' is not referencd but not defined in the file '{1}'", dep.Name, package.FilePath));
+                        throw new ArgumentException(string.Format("'{0}' is referencd but not defined in the file '{1}'", dep.Name, package.FilePath));
```

Comment:
@rkeithhill : yes, I'm well aware of that, as I noted in the commit message / pull request description. I'd really like to avoid mixing that in with this PR.

My goal in splitting this commit out is to make it easier for people to understand what is actually a functional change in your UX as opposed to things which are really minor and shouldn't change meaning.